src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
+    align: left;
+    bottom: 0;
+    background-color: ${colorMap.greyDarkThree};
+    display: inline-block;
+    font-family: 'Akzidenz-Grotesk';
```

Comment:
```suggestion
    font-family: akzidenz;
```
This is how we have it in `font.css` so that will get the right font in there.

Reply:
Oops! Sorry, totally forgot that you asked me to look into that file